books/kestrel/number-theory/tonelli-shanks.lisp

```diff
@@ -185,25 +188,24 @@
 ;; that can't happen)
 
 ;; Future work: prove correctness; improve guards
-
+;; "p must be an odd prime"
```

Comment:
This comment should go  in the :long doc.  "p must be prime." --> "p must be an odd prime"